invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/AccessRequestForm.js

```diff
@@ -22,10 +22,12 @@ export class AccessRequestForm extends Component {
       loading: false,
       error: undefined,
       modalOpen: false,
+      retry: true,
     };
+    // nullable().required() is needed to avoid ``null`` type message errors.
     this.accessRequestSchema = Yup.object({
-      email: Yup.string().email().required(),
-      full_name: Yup.string().required(),
+      email: Yup.string().email().nullable().required(),
```

Comment:
When nothing was inserted, `yup` showed a message "email must be a string, but the final value was null". We can make it `nullable` and `required` to force only the `required` validation